[2025-07-20T19:51:06.364+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-20T19:51:06.390+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_weather_forecast.fetch_forecast manual__2025-07-20T19:51:04.761816+00:00 [queued]>
[2025-07-20T19:51:06.397+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_weather_forecast.fetch_forecast manual__2025-07-20T19:51:04.761816+00:00 [queued]>
[2025-07-20T19:51:06.398+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-20T19:51:06.408+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_forecast> on 2025-07-20 19:51:04.761816+00:00
[2025-07-20T19:51:06.416+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fetch_weather_forecast', 'fetch_forecast', 'manual__2025-07-20T19:51:04.761816+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/fetch_weather_data.py', '--cfg-path', '/tmp/tmpms5dp1hl']
[2025-07-20T19:51:06.419+0000] {standard_task_runner.py:91} INFO - Job 35: Subtask fetch_forecast
[2025-07-20T19:51:06.419+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=185) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-20T19:51:06.420+0000] {standard_task_runner.py:63} INFO - Started process 187 to run task
[2025-07-20T19:51:06.462+0000] {task_command.py:426} INFO - Running <TaskInstance: fetch_weather_forecast.fetch_forecast manual__2025-07-20T19:51:04.761816+00:00 [running]> on host 74203d08dcb8
[2025-07-20T19:51:06.538+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_weather_forecast' AIRFLOW_CTX_TASK_ID='fetch_forecast' AIRFLOW_CTX_EXECUTION_DATE='2025-07-20T19:51:04.761816+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-20T19:51:04.761816+00:00'
[2025-07-20T19:51:06.539+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-20T19:51:07.066+0000] {logging_mixin.py:188} INFO - Failed for Ho Chi Minh: 429
[2025-07-20T19:51:08.008+0000] {logging_mixin.py:188} INFO - Failed for Ha Noi: 429
[2025-07-20T19:51:08.946+0000] {logging_mixin.py:188} INFO - Failed for Da Nang: 429
[2025-07-20T19:51:09.926+0000] {logging_mixin.py:188} INFO - Failed for Hue: 429
[2025-07-20T19:51:10.872+0000] {logging_mixin.py:188} INFO - Failed for Hai Phong: 429
[2025-07-20T19:51:11.831+0000] {logging_mixin.py:188} INFO - Failed for Can Tho: 429
[2025-07-20T19:51:12.789+0000] {logging_mixin.py:188} INFO - Failed for Bien Hoa: 429
[2025-07-20T19:51:13.721+0000] {logging_mixin.py:188} INFO - Failed for Quang Ninh: 429
[2025-07-20T19:51:14.659+0000] {logging_mixin.py:188} INFO - Failed for Nghe An: 429
[2025-07-20T19:51:15.599+0000] {logging_mixin.py:188} INFO - Failed for Ninh Binh: 429
[2025-07-20T19:51:16.110+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-20T19:51:16.111+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-20T19:51:16.119+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=fetch_weather_forecast, task_id=fetch_forecast, run_id=manual__2025-07-20T19:51:04.761816+00:00, execution_date=20250720T195104, start_date=20250720T195106, end_date=20250720T195116
[2025-07-20T19:51:16.161+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-20T19:51:16.180+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-20T19:51:16.182+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
